### PR TITLE
Added support for UIkits's sortable component.

### LINKED ISSUE #24: Add Sortable Component
I made a component today for UIkit's sortable component. I still need to add some more tests and finish the template for the demo site. Before I go too far with the demo site, I'd like some feedback on my approach to adding the component specific events and attributes. I noticed the evented mixin, it didn't seem right to add component specific events there. I'm sure I didn't follow some of your style conventions.

Anyway, I'll make whatever style/design adjustments and work on the demo page if y'all are interested in a PR.

Here's the commit to my fork: https://github.com/jayvarner/ember-uikit/commit/c42851356e300b6667fa3ef576d66fd98da9fbf3

### PR DESCRIPTION
**Pull request by jayvarner**
Resolves #24. I'm happy to address any lingering style issues that I missed. I did leave out the "demo" column on the dummy app due to previously discussed updating issues.

### CODE REVIEW

File: tests/dummy/app/controllers/components/sortable.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import Controller from '@ember/controller';
+import { A } from '@ember/array';
+import UIkit from 'uikit';
+
+export default Controller.extend({
+  firedEvent: null,
+  gAMoved: false,
+  // items: computed(function() {
+  //   return ;
+  // }).property('items'),
```
Comment (anehx) [inline tests/dummy/app/controllers/components/sortable.js]: Obsolete?

File: tests/dummy/app/templates/application.hbs [hunk 2]
```diff
@@ -13,7 +13,8 @@
             <li>{{#link-to 'components.icon'}}Icon{{/link-to}}</li>
             <li>{{#link-to 'components.label'}}Label{{/link-to}}</li>
             <li>{{#link-to 'components.list'}}List{{/link-to}}</li>
-            <li>{{#link-to 'components.spinner'}}Spinner{{/link-to}}</li>
+             <li>{{#link-to 'components.sortable'}}Sortable{{/link-to}}</li>
+           <li>{{#link-to 'components.spinner'}}Spinner{{/link-to}}</li>
```
Comment (anehx) [inline tests/dummy/app/templates/application.hbs]: Please fix the indent.

File: tests/dummy/app/templates/components/sortable.hbs [hunk 3]
```diff
@@ -0,0 +1,204 @@
+<h2>Sortable</h2>
+<hr />
```
Comment (anehx) [inline tests/dummy/app/templates/components/sortable.hbs]: The indentation and style on this page is not the same as in all the others.. Please style it exactly like the others.

File: tests/dummy/app/templates/components/sortable.hbs [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+            <tr>
+              <td><code>threshold</code></td>
+              <td><code>Number</code></td>
+              <td><code>10</code></td>
+              <td>Mouse move threshold before dragging starts.</td>
+            </tr>
+            <tr>
+              <td><code>clsItem</code></td>
+              <td><code>String</code></td>
+              <td><code>uk-sortable-item	</code></td>
```
Comment (anehx) [inline tests/dummy/app/templates/components/sortable.hbs]: Whitespaces

File: tests/dummy/app/templates/components/sortable.hbs [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+            <tr>
+              <td><code>clsNoDrag</code></td>
+              <td><code>String</code></td>
+              <td><code>uk-sortable-nodrag</code></td>
+              <td>Disable dragging on elements with this class.</td>
+            </tr>
+            <tr>
+              <td><code>clsEmpty</code></td>
+              <td><code>String</code></td>
+              <td><code>uk-sortable-empty</code></td>
```
Comment (anehx) [inline tests/dummy/app/templates/components/sortable.hbs]: If the default values are strings, use quotes to mark them as such

**Comment by jayvarner** [comment]
OK, I cleaned up the whitespace and removed unused code in the dummy app. I have a few more in the works. Hopefully those will go smoother.

**Pull request merged**